src/results/ResultsContainer.tsx

```diff
@@ -120,38 +121,48 @@ export class Results extends Component<IResultsProps, ResultsContainerState> {
       dispatchAddFacet,
       dispatchRemoveFacet,
       dispatchUpdateColumnSort,
+      namespace,
       sort,
     } = this.props;
     const { selectedRows } = this.state;
     if (isFetching) {
       return <h3>Loading...</h3>;
     }
+    const info = infoMappings[namespace];
     return (
-      <SideBarLayout
-        sidebar={
-          <Facets
-            data={facets}
-            selectedFacets={selectedFacets}
-            addFacet={dispatchAddFacet}
-            removeFacet={dispatchRemoveFacet}
-          />
-        }
-        content={
-          <ResultsTable
-            results={results}
-            columnNames={columns}
-            handleRowSelect={this.handleRowSelect}
-            selectedRows={selectedRows}
-            handleHeaderClick={dispatchUpdateColumnSort}
-            sort={sort}
-          />
-        }
-      />
+      <Fragment>
+        <SideBarLayout
+          title={
+            <PageIntro title={info.name} links={info.links}>
```

Comment:
I think it will be good to have further discussion about passing children via props like this. I think the community convention here is to prepare the child element in the function scope and then pass it like `<PageIntroElement />` rather creating the child element inline when it is passed as a prop.

Reply:
Let's have a chat